### PR TITLE
Add EVA profile for MPI-EVA (clean)

### PR DESCRIPTION
**Pull request by jfy133**
Currently only for DAG.

Adds eager specific pipeline too.

Sorry for commit mess. Java sucks.

### CODE REVIEW

File: conf/eva.config [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+profiles {
+    archgen {
+      params {
+        igenomes_base = "/projects1/public_data/igenomes/"
+        config_profile_description = 'MPI-EVA archgen profile, provided by nf-core/configs.'
+        max_memory = 256.GB
+        max_cpus = 32
+        max_time = 720.h
+        //Illumina iGenomes reference file path
+        igenomes_base = "/projects1/public_data/igenomes/"
```
Comment (apeltzer) [inline conf/eva.config:35]:
```suggestion
```

Duplicate of the above

Comment (apeltzer) [reply conf/eva.config:35]: See 6 lines above

File: docs/pipeline/eager/mpcdf.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# nf-core/configs: mpcdf eager specific configuration
```
Comment (apeltzer) [inline docs/pipeline/eager/mpcdf.config:1]: This isn't EVA anymore ;-)

**Review by apeltzer — CHANGES_REQUESTED** [review]
Coupl of points

File: docs/pipeline/eager/mpcdf.config (hunk 2 above)
Comment (jfy133) [reply docs/pipeline/eager/mpcdf.config:1]: Ah yeah, this is a bonus because apparently I forgot to commit it in the previous profile PR :eyes:

**Review by apeltzer — APPROVED** [review]

**Pull request merged**